src/sider/frisktilarbeid/FattVedtakSkjema.tsx

```diff
@@ -163,6 +166,12 @@ export default function FattVedtakSkjema() {
               label={texts.begrunnelse.label}
               error={errors.begrunnelse?.message}
             />
+            <Alert variant="info" className="[&>*]:max-w-fit">
+              <Heading spacing size="small" level="3">
```

Comment:
Testet litt lokalt, og synes `size="small"` på alerten og `size="xsmall"` på headingen føltes mer enn stort nok. Vet ikke hvor grundig du og Klaudia og Stine har sett på dette? 😊
```suggestion
            <Alert variant="info" contentMaxWidth={false} size="small">
              <Heading spacing size="xsmall" level="3">
```